### PR TITLE
Library page New Book Button[PAR-52]

### PR DESCRIPTION
**Pull request by Gatlinn**

### CODE REVIEW

File: src/LibraryPage.tsx [hunk 1]
```diff
@@ -45,48 +61,53 @@ const LibraryPage: React.FC = () => {
...
-          },
-          owner_contact: "",
-          requester_contact: "",
-          accept_date: new Date(),
-          request_date: new Date(),
-          loan_start_date: new Date(),
-          loan_end_date: new Date(),
+
+    console.log(user);
+  }, [user]);
```
Comment (lemniscate8) [inline src/LibraryPage.tsx:65]: `console.log` and useEffect leftover from testing

Comment (Hashiphoto) [inline src/LibraryPage.tsx:65]: Probably forgot to remove the console.log here from testing

File: src/LibraryPage.tsx [hunk 2]
```diff
@@ -45,48 +61,53 @@ const LibraryPage: React.FC = () => {
...
+  useEffect(() => {
+    fetch(`http://paralibrary.digital/api/libraries/`, {
```
Comment (Hashiphoto) [inline src/LibraryPage.tsx]: Can you remove the trailing slash (/) for consistency?

File: src/LibraryPage.tsx [hunk 3]
```diff
@@ -45,48 +61,53 @@ const LibraryPage: React.FC = () => {
...
+    })
+      .then((res) => {
+        return res.json();
+      })
+      .then(
+        (result) => {
+          const lib = toLibrary(result);
+          setBooks(lib.books);
+          setUser(lib.user);
+          console.log(user);
```
Comment (Hashiphoto) [inline src/LibraryPage.tsx]: Probably remove this also

**Review by Hashiphoto — APPROVED** [review]
Looks good! A few very minor suggestions

**Review by lemniscate8 — APPROVED** [review]
Just a few things like `console.log`s and some imports that might not be necessary that I'd take out. Other than that looks good to me!

**Pull request merged**